### DISCUSSION TITLE
Sites API not working

### DISCUSSION THREAD

**Discussion by ajaytank1991** [plausible/analytics#4544] [Self-Hosted Support]
Hello,

I can not able to access the sites api

I have try to update scopes, but still I can not able to access sites api.

UPDATE api_keys SET scopes = '{sites:provision:*}' WHERE name = 'Development';

is there any other method for do it? 

Please assist me about it.

**Comment by ruslandoga** [comment]
👋 @ajaytank1991 

Sites management API is not included in Plausible CE.

**Reply by abejr** [reply]
Hello @ruslandoga,
Are there any plans to include this feature in the future (in the community edition)? Additionally, is it possible to add or update sites directly in the PostgreSQL database table "sites"?

**Reply by ruslandoga** [reply]
👋 @abejr 

I am not aware of any such plans :)

> Additionally, is it possible to add or update sites directly in the PostgreSQL database table "sites"?

It's possible. Note that you would also need to create memberships between sites and users.

**Reply by abejr** [reply]
@ruslandoga thank you!
